src/components/templates/Auditorium/components/Section/Section.scss

```diff
@@ -0,0 +1,105 @@
+@import "scss/constants";
+
+// If you change this, make sure to also change it in constants.ts
+$seat-size: 4vh;
+$seat-size-min: 25px;
+$seat-spacing: 3px;
+
+$spacing: calc(1 * min(var(--seat-size), var(--seat-size-min)));
+
+.section {
+  --seat-size: #{$seat-size};
+  --seat-size-min: #{$seat-size-min};
+  --seat-spacing: #{$seat-spacing};
+
+  background-attachment: fixed;
+  background-position: center 46%;
```

Comment:
Why `46%`?

Reply:
Was copy pasted from Audience, but I have centered it

Reply:
fixed here https://github.com/sparkletown/sparkle/pull/1251/commits/adf6f1b18663d0e43209d78822eb78a1f85af80d